squape/video.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023, Cyber Alpaca
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+import os
+from contextlib import contextmanager
+
+import squishinfo
```

Comment:
squishinfo was not used before, thus it's not mentioned in the pre-commit configuration yet.
Please add it into pre-commit-config.yaml file as:
`--unclassifiable-application-module, squishinfo,`

Reply:
Added with [ad1cf3d](https://github.com/CyberAlpaca/squish-api-python-extension/pull/16/commits/ad1cf3de32521e92df6f1ff55b460d199652dfdb)